explore/src/main/scala/explore/config/sequence/SequenceTileHelper.scala

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.config.sequence
+
+import cats.Eq
+import cats.derived.*
+import cats.effect.IO
+import cats.effect.Resource
+import cats.syntax.all.*
+import clue.StreamingClient
+import crystal.Deglitcher
+import crystal.Pot
+import crystal.react.*
+import crystal.react.hooks.*
+import explore.*
+import explore.model.AppContext
+import explore.model.Observation
+import japgolly.scalajs.react.*
+import lucuma.core.enums.ObserveClass
+import lucuma.core.enums.StepStage
+import lucuma.core.math.SignalToNoise
+import lucuma.core.model.Target
+import lucuma.core.model.sequence.InstrumentExecutionConfig
+import lucuma.schemas.ObservationDB
+import lucuma.schemas.model.ExecutionVisits
+import lucuma.schemas.odb.SequenceQueriesGQL.*
+import lucuma.schemas.odb.input.*
+import queries.common.ObsQueriesGQL
+import queries.common.TargetQueriesGQL
+import queries.common.VisitQueriesGQL
+import queries.common.VisitQueriesGQL.*
+
+import scala.concurrent.duration.*
+
+trait SequenceTileHelper:
+  protected case class SequenceData(
+    config:     InstrumentExecutionConfig,
+    snPerClass: Map[ObserveClass, SignalToNoise]
+  ) derives Eq
+
+  protected object SequenceData:
+    def fromOdbResponse(data: SequenceQuery.Data): Option[SequenceData] =
+      data.observation.flatMap: obs =>
+        obs.execution.config.map: config =>
+          SequenceData(
+            config,
+            Map.empty
+            // (
+            //   ObserveClass.Science     -> obs.itc.science.selected.signalToNoise,
+            //   ObserveClass.Acquisition -> obs.itc.acquisition.selected.signalToNoise
+            // )
+          )
+
+    given Reusability[SequenceData] = Reusability.byEq
+  end SequenceData
+
+  import cats.effect.Temporal
+  extension [F[_]: Temporal, A](stream: fs2.Stream[F, A])
+    def throttle(duration: FiniteDuration): fs2.Stream[F, A] =
```

Comment:
Isn't this extension in `extensions.scala`, too?

Reply:
Yup, forgot to remove it when I moved it there.

Anyway, I'm deleting from both places since we are not using it after all.